scripts/dashboard/build-dashboard.js

```diff
@@ -142,7 +142,7 @@ async function start() {
     ]);
     writeToFile({ hotDiscussions, goodFirstIssues });
   } catch (e) {
-    writeToFile({ hotDiscussions: [], goodFirstIssues: [] });
```

Comment:
I have a question, why is the script getting errors on every 2nd-3rd iteration and catch block is implemented?

Reply:
Sometimes GitHub API returns internal server error and sometimes it says that you have reached your temporary limit (even if there is points remaining for that hour.)

I couldn't figure out what exactly the problem was.

Reply:
yeah, we are making quite some API calls to gather data 😅

Reply:
It can be possible that you are making multiple GET requests in a single second using the script, so it will exceed the Github API per second limit. You can better make the script sleep for 1-2 seconds if needed, to decrease the API calls in script per second.

Reply:
I agree this can be improved with simple delay or retry mechanism. Octokit had plugins for retry but for REST client, maybe they already have it for GraphQL.

Anyway I suggest ☝🏼 is solved as a followup because current bug is not nice and dashboard is empty again